### ISSUE TITLE
J behaviour

### ISSUE THREAD

**Issue by frenchjam** [PsyPhy/ProjectDiskGun#8]
I am trying to understand the behavior associated with the J key. In the so-called GUI state, it appears that the 2D crosshairs are drawn to aid in aiming. In the 'turning gui' state, the gui element is made to follow the rotation of the gun, but the crosshair texture is not drawn.  I suspect that this is work in progress. Can this be clarified?

**Comment by mikha42** [comment]
Yes, i initially planed (Before giithub) to make a state, toggled with a key press, that rotates the crosshair (_edit_ It would set the gun rotation to the crosshair). this is because the gun was initially un-toggleable. But i stopped making it because as i rotated the GUI it didn't seem to rotate the crosshair, but instead it moved the turning indicator.

I should remember to remove excess coding that i make.
